cmd/kubeadm/app/preflight/checks.go

```diff
@@ -104,7 +104,7 @@ func (criCheck CRICheck) Check() (warnings, errors []error) {
 		errors = append(errors, fmt.Errorf("unable to find command crictl: %s", err))
```

Comment:
[3]
outside of the diff but maybe we can change `%s` to `%v`.

Reply:
as this is cherry-pick to stable branch for specific issue, I'd suggest to do changes %s -> %v as separate PR.